### PR TITLE
Styles were updated

### PR DESCRIPTION
**Pull request by mgrinko**
[Preview is here](https://mate-academy.github.io/git-webinar/2/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Git Webinar</title>
+  <style>
+    section {
+      border: 2px solid blue;
+      border-radius: 5px;
+      background-color: burlywood;
```
Comment (mgrinko) [inline index.html]: WHy did you change the color

**Comment by doroshenkoSaac** [comment]
first comment

File: index.html [hunk 2]
```diff
@@ -5,6 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Git Webinar</title>
+  <style>
+    section {
+      border: 2px solid blue;
```
Comment (mgrinko) [inline index.html:10]: sadfasdfasdf

Comment (mgrinko) [reply index.html:10]: asdsadfsdfsdf

File: index.html [hunk 3]
```diff
@@ -5,6 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Git Webinar</title>
+  <style>
```
Comment (mgrinko) [inline index.html:8]: asdsadasdfsdf

File: index.html [hunk 4]
```diff
@@ -5,6 +5,13 @@
...
+    section {
+      border: 2px solid blue;
+      border-radius: 5px;
```
Comment (ghost) [inline index.html:11]: I think border radius is too big

File: index.html (hunk 1 above)
Comment (mgrinko) [reply index.html]: fixed
